engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -506,24 +526,91 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         spawnPos.add(offset);
 
         BlockFamily block = blockManager.getBlockFamily(blockName);
-        if (block == null) {
-            return "";
-        }
 
         BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
         EntityRef blockItem = blockItemFactory.newInstance(block);
 
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "Drops many blocks as per value choosen by the player", helpText = "the particular block can be seleted which the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDropper(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
+
+        //This is a loop which gives the particular amount of block the player wants to spawn
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+        offset.scale(3);
+        spawnPos.add(5,10,0);
+        BlockFamily block = blockManager.getBlockFamily(blockName);
+        if (block == null) {
+            return "";
```

Comment:
In case of an error please return a useful message to the user, that'll show up in the console to indicate something went wrong, hopefully with some details :-)